TSP/DAL/DAL.csproj

```diff
@@ -1,15 +1,17 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
+    <Nullable>enable</Nullable>
   </PropertyGroup>
 
   <ItemGroup>
 
     <PackageReference Include="EntityFramework" Version="6.4.4" />
```

Comment:
Is it needed here?

Reply:
I only found this way to enable this. What should i do to solve this problem?

Reply:
fix